soccer_vision_msgs/msg/Robot.msg

```diff
@@ -0,0 +1,28 @@
+# Describes a detected obstacle from vision modules, in image coordinates.
+
+uint8 NUMBER_UNKNOWN=0
+
+uint8 TEAM_UNKNOWN=0
+uint8 TEAM_OWN=1
+uint8 TEAM_OPPONENT=2
+
+# A bounding box that surrounds the obstacle in the image, this does not have to include
+# the whole obstacle, but only the part of the obstacle that was detected.
+vision_msgs/BoundingBox2D bb
+
+# The number of the robot, if it can be read from the image.
+# Values can range from NUMBER_UNKNOWN(0) up to 255.
+uint8 player_number
+
+# Which team's robot it is. 
+# Value can be TEAM_UNKNOWN, TEAM_OWN or TEAM_OPPONENT
```

Comment:
In this case, the responsibility to map the team colors to the teams is moved to the vision, but I kind of like it since it is more general if you consider other factors such as visual differences between the robot platforms in the classification.